### PR TITLE
Added a command-line option that will force all tests to be flaky

### PR DESCRIPTION
**Pull request by wsong**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @wsong, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by wsong** [comment]
CLA signed.

**Comment by boxcla** [comment]
Verified that @wsong has just signed the CLA. Thanks, and we look forward to your contribution.

File: flaky/_flaky_plugin.py [hunk 1]
```diff
@@ -186,6 +186,45 @@ def add_report_option(add_option):
...
+        )
+        add_option(
+            '--max-runs',
+            action="store",
+            dest="max_runs",
+            type="int",
+            default=None,
+            help="If --force-flaky is specified, we will run each test at "
+                 "most this many times (unless the test has its own flaky "
+                 "decorator."
```
Comment (jmoldow) [inline flaky/_flaky_plugin.py]: Missing closing parenthesis.

File: flaky/_flaky_plugin.py [hunk 2]
```diff
@@ -186,6 +186,45 @@ def add_report_option(add_option):
...
+        )
+        add_option(
+            '--min-passes',
+            action="store",
+            dest="min_passes",
+            type="int",
+            default=None,
+            help="If --force-flaky is specified, we will run each test least "
+                 "most this many times (unless the test has its own flaky "
+                 "decorator."
```
Comment (jmoldow) [inline flaky/_flaky_plugin.py]: Misworded, and missing closing parenthesis.

File: flaky/_flaky_plugin.py [hunk 3]
```diff
@@ -349,3 +407,35 @@ def _get_test_method_and_name(cls, test):
...
+            `int`
+        :param min_passes:
+            The value of the FlakyNames.MIN_PASSES attribute to use.
+        :type min_passes:
+            `int`
+        """
+        if max_runs is None:
+            max_runs = 2
+        if min_passes is None:
+            min_passes = 1
```
Comment (jmoldow) [inline flaky/_flaky_plugin.py]: Why not just use default values of 2 and 1 for the ``max_runs`` / ``min_passes`` params?

Comment (wsong) [inline flaky/_flaky_plugin.py]: Done.

**Comment by Jeff-Meadows** [comment]
Hi Wayne, thanks for the PR. The change looks good overall - could you fix up the pylint warnings (you can check them from the TravisCI build).

I think the repeated code warning could be solved by having the flaky decorator use your new make_method_flaky method or by making that method apply the decorator.

**Comment by wsong** [comment]
Importing _flaky_plugin in flaky_decorator seemed somewhat hacky (and vice-versa), so I created a new module.  Please take another look.

**Comment by Jeff-Meadows** [comment]
Makes sense to me. Please fix up the other pylint warnings and it looks good to merge!

**Comment by wsong** [comment]
Done.

**Comment by Jeff-Meadows** [comment]
Thanks! I'll create a new version this weekend so your change can be pip installed.

**Comment by wsong** [comment]
Thanks!

**Pull request merged**